### PR TITLE
15.goroutines

### PR DESCRIPTION
**Pull request by evgeniiburdin**

### CODE REVIEW

File: course1/15.goroutines/1.goroutines.run/task1.15.1.1/main.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+
+	"time"
```
Comment (inadenenko) [inline course1/15.goroutines/1.goroutines.run/task1.15.1.1/main.go:5]: group standard imports

File: course1/15.goroutines/3.goroutines_chan/task1.15.3.2/main.go [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+		statusCode := <-result
+		if statusCode != 200 {
+			panic(fmt.Sprintf("Ошибка при отправке запроса: %d", statusCode))
+		}
+	}
+
+	fmt.Println("Все горутины завершили работу")
+}
+
+func benchRequest(url string, parallelRequest int, requestCount int) <-chan int {
```
Comment (inadenenko) [inline course1/15.goroutines/3.goroutines_chan/task1.15.3.2/main.go]: rename `parrallelRequest` variable

File: course1/15.goroutines/6.goroutine_sync/task1.15.6.5/main.go [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+func keyBuilder(keys ...string) string {
+	return strings.Join(keys, ":")
+}
+
+// GetUser converts interface{} type to *User type. The object retrieved from the cache is expected to be of type *User.
+func GetUser(i interface{}) *User {
+	return i.(*User)
+}
+
+// main creates a new Cache object, runs 100 goroutines adding 100 `User` objs to the Cache, runs 100 goroutines
```
Comment (inadenenko) [inline course1/15.goroutines/6.goroutine_sync/task1.15.6.5/main.go:58]: why so many comments in a file?

Comment (evgeniiburdin) [reply course1/15.goroutines/6.goroutine_sync/task1.15.6.5/main.go:58]: there was a requrement in the task to give the code a detailed explanation...

**Review by inadenenko — APPROVED** [review]

**Pull request merged**